src/controllers/attachment/index.ts

```diff
@@ -0,0 +1,94 @@
+import { UploadedFile, Request, Controller, Get, Route, Path, Post, Header, Response, SuccessResponse } from 'tsoa'
+import { Logger } from 'pino'
+import express from 'express'
+
+import { logger } from '../../lib/logger'
+import Database, { Models, Query } from '../../lib/db'
+import type { Attachment } from '../../models'
+import { BadRequest, NotFound } from '../../lib/error-handler'
+
+type File = {
+  [k: string]: string
+}
+
+@Route('attachment')
```

Comment:
still missing the @Tags to give its own section in swagger

Reply:
Yea, it was defaulting to 'default' added a @Tags decorator.
![Screenshot 2023-03-17 at 13 07 36](https://user-images.githubusercontent.com/11794402/225913369-e8f37421-f60d-4289-bc5b-3f35bf536694.png)